spa/src/pages/ConfigSteps/index.tsx

```diff
@@ -221,10 +221,12 @@ const ConfigSteps = () => {
 		try {
 			analyticsClient.sendUIEvent({ actionSubject: "connectOrganisation", action: "clicked" }, { mode });
 			const connected = await AppManager.connectOrg(gitHubInstallationId);
-			analyticsClient.sendTrackEvent({ actionSubject: "organisationConnectResponse", action: connected ? "success" : "fail"}, { mode });
 			if (connected instanceof AxiosError) {
-				showError(modifyError(connected, { orgLogin }, { onClearGitHubToken: clearGitHubToken, onRelogin: reLogin }));
+				const errorObj = modifyError(connected, { orgLogin }, { onClearGitHubToken: clearGitHubToken, onRelogin: reLogin });
```

Comment:
\nit
We can already grab the errorCode from `connected`, as it is an instance of `AxiosError`. 
We can avoid returning the `errorCode` from `modifyError`.

Reply:
Hmm to get the errorCode, we need to do this `errorCode = connected?.response?.data?.errorCode || "UNKNOWN";` .  I think I'd rather use the errorObj instead.